api/destination.go

```diff
@@ -6,21 +6,21 @@ import (
 )
 
 type Destination struct {
-	ID         uid.ID                `json:"id"`
-	UniqueID   string                `json:"uniqueID" form:"uniqueID" example:"94c2c570a20311180ec325fd56"`
-	Name       string                `json:"name" form:"name"`
-	Kind       string                `json:"kind"`
+	ID         uid.ID                `json:"id" note:"ID of the destination" example:"7a0b06b30F"`
+	UniqueID   string                `json:"uniqueID" form:"uniqueID" note:"Unique ID generated by the connector" example:"94c2c570a20311180ec325fd56"`
+	Name       string                `json:"name" form:"name" note:"Name of the destination" example:"production-cluster"`
+	Kind       string                `json:"kind" note:"Kind of destination. eg. kubernetes or ssh or postgres" example:"kubernetes"`
 	Created    Time                  `json:"created"`
 	Updated    Time                  `json:"updated"`
-	Connection DestinationConnection `json:"connection"`
+	Connection DestinationConnection `json:"connection" note:"Object that includes the URL and CA for the destination" example:"{'url':'localhost:8443', 'ca':'clustercertificate'}"`
```

Comment:
this example should probably be covered by the DestinationConnection struct

Reply:
that doesn't show up in the openapi json doc